tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodMetadataInfo.cs

```diff
@@ -27,6 +36,20 @@ public MethodMetadataInfo(string[] parameterNames, string[] localVariableNames,
         /// </summary>
         public string[] LocalVariableNames { get; }
 
+        /// <summary>
+        /// Gets the names and the values of the async method's local variable from the hoist object - i.e. the state machine.
+        /// May contains fields that they not locals in the kick off method, ATM we skip only the `builder`, `state` and `this` fields
```

Comment:
```suggestion
        /// May contains fields that are not locals in the async kick-off method, ATM we skip only the `builder`, `state` and `this` fields
```

Reply:
Fixed in 871a652d9b150d63443d87a542e34d4643145adf